internal/pkg/service/cli/cmd/remote/create/branch/cmd.go

```diff
@@ -23,7 +23,7 @@ type Flags struct {
 	StorageAPIHost  configmap.Value[string] `configKey:"storage-api-host" configShorthand:"H" configUsage:"if command is run outside the project directory"`
 	StorageAPIToken configmap.Value[string] `configKey:"storage-api-token" configShorthand:"t" configUsage:"storage API token from your project"`
 	Name            configmap.Value[string] `configKey:"name" configShorthand:"n" configUsage:"name of the new branch"`
-	OutputJSON      configmap.Value[string] `configKey:"output-json" configUsage:"output as JSON file"`
+	OutputJSON      configmap.Value[string] `configKey:"output-json" configUsage:"output as a JSON file"`
```

Comment:
it should match the documentation. [e334e47](https://github.com/keboola/developers-docs/pull/308/commits/e334e47d60938a0b969993cc396faefa2e765c63)